scripts/completion-tests/test-completion.sh

```diff
@@ -34,10 +34,26 @@ export -f set_shell_debug_level
 set_shell_debug_level 2
 
 BINARY_NAME=helm
-BINARY_ROOT=${ROBOT_HELM_PATH:-${SCRIPT_DIR}/../../../helm/bin}
-BINARY_PATH_DOCKER=${BINARY_ROOT}/../_dist/linux-amd64
+HELM_DIR=${SCRIPT_DIR}/../../.acceptance
+HELM_REPO=${HELM_DIR}/helm
+BINARY_ROOT=${HELM_REPO}/bin
+BINARY_PATH_DOCKER=${HELM_REPO}/_dist/linux-amd64
 BINARY_PATH_LOCAL=${BINARY_ROOT}
 
+if [ -d ${HELM_REPO} ]; then
+    (cd ${HELM_REPO} && make build build-cross) &> /dev/null
+    exit_code=$?
+    if [ $exit_code -ne 0 ]; then
+        exit $exit_code;
+    fi
+else
+    (cd ${HELM_DIR} && git clone https://github.com/helm/helm.git && cd helm && make build build-cross) &> /dev/null
```

Comment:
We should only clone one level of depth to clone faster

Reply:
yes you are right we could do that, nice suggestion, and one more issue I thought could be what if a person wants to run this test on any specified branch of the helm repo then he would have to modify this line. So instead of cloning what if we provide a direct path of bin here, anyway [this is](https://github.com/helm/acceptance-testing/blob/master/scripts/completion-tests/test-completion.sh#L38) a path of `linux-amd64/bin`. It will be a small, efficient and elegant fix and we won't need cloning repo anymore.